frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentStepper.tsx

```diff
@@ -5,15 +5,18 @@ import { PublicEnrollmentFormStep } from 'enums/publicEnrollment';
 
 export const PublicEnrollmentStepper = ({
   activeStep,
+  includePaymentStep,
 }: {
   activeStep: PublicEnrollmentFormStep;
+  includePaymentStep: boolean;
 }) => {
   const { t } = usePublicTranslation({
     keyPrefix: 'vkt.component.publicEnrollment.stepper',
   });
 
-  const getStepNumbers = Object.values(PublicEnrollmentFormStep)
+  const stepNumbers = Object.values(PublicEnrollmentFormStep)
     .filter((i) => !isNaN(Number(i)))
+    .filter((i) => includePaymentStep || i < PublicEnrollmentFormStep.Done)
```

Comment:
Syy siihen miksi täällä vertailu tehdään Done-steppiä vasten ja alempana Pay-steppiä vasten liittyy siihen, että näin voitaisiin koodia muuttamatta lisätä nähdäkseni teoriassa noiden väliin myös lisästeppejä jos olisi jostain syystä tarvis